bson/_cbsonmodule.c

```diff
@@ -70,6 +70,11 @@ struct module_state {
     PyObject* _bid_str;
     PyObject* _replace_str;
     PyObject* _astimezone_str;
+    PyObject* _id_str;
+    PyObject* _$ref_str;
```

Comment:
While these are valid identifiers, they are a bit confusing (and also tripped up the syntax highlighting), can we remove the dollar signs?

Reply:
yes, fixed!